### DISCUSSION TITLE
Invocation the original command and man page of dd after installing coreutils with brew on macOS Catalina.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [Homebrew/discussions#1062] [Everyday usage]
On macOS Catalina, I installed the coreutils with `brew install coreutils` and set the following PATH in `~/.zshrc`:

`export PATH="$(brew --prefix coreutils)/libexec/gnubin:$PATH"`

But with the above setting, how can I call the original man page and command of `dd` shipped with macOS conveniently?

Regards,
HY

**Comment by carlocab** [comment]
This is part of the reason why `gnubin` isn't linked into the prefix. One solution is to not add `gnubin` to your `PATH`, and just call the `coreutils` binaries with the `g` prefix where necessary.

Alternatively, you could try
```
man /usr/share/man/man1/dd.1
```
If you want to us `/bin/dd`, you will also need to call it by its full path (i.e. `/bin/dd`), as unqualified calls to `dd` will pick up `coreutils`'s `dd`.

**Reply by gromgit** [reply]
That's all correct, except:
>```
>man /usr/share/man/man1/dd.1
>```

isn't necessary. `man dd` should _always_ give you the system `dd` man page, because the coreutils man page's filename is `gdd.1`, and it's not aliased to `dd.1` in any way.

This is another reason to reference coreutils binaries with the `g` prefix, rather than adding the `gnubin` directory to `PATH`. That way, `man XYZ` always gives you the man page corresponding to `XYZ`, so there will never be any documentation confusion.

**Reply by hongyi-zhao** [reply]
Got it. Thank you everyone for your responses.
